spec/lib/find_by_path_spec.rb

```diff
@@ -1,15 +1,15 @@
 require "rails_helper"
 
-describe FindByPath do
-  # Using an actual Model as it's a real pain to mock mongoid criterias and
-  # similar
-  class CompatibleModel
-    include Mongoid::Document
-    field :base_path, type: String
-    field :routes, type: Array, default: []
-    field :redirects, type: Array, default: []
-  end
+# Using an actual Model as it's a real pain to mock mongoid criterias and
+# similar
+class CompatibleModel
+  include Mongoid::Document
+  field :base_path, type: String
+  field :routes, type: Array, default: []
+  field :redirects, type: Array, default: []
+end
```

Comment:
This could also be moved into the class, but I think it'll need to be paired with a `stub_const("CompatibleModel", compatible_model_class)` that runs in a before block.

Reply:
Having difficulty with this one, because `spec/support/factory_bot_spec.rb` also needs the const to be available

Reply:
You're right. This test is really rather odd and sadly has my face all over the Git blame /shame

I had a quick go at cutting FactoryBot out of the equation here: https://github.com/alphagov/content-store/commit/49acd5cd8cc2a90980a40ca586ee92488a0ffc19

It maybe fixes 20% of the oddness of the tests mind 😆  the rest will have to wait for another day.